### PR TITLE
Updates play icon

### PR DESCRIPTION
**Pull request by duarte**
Updates play icon to remove the words 'Watch now' from the button.
Also changes hover effect to the button scale effect, and adds SVG play icon arrow.

### CODE REVIEW

File: src/css/_youtube.scss [hunk 1]
```diff
@@ -24,14 +23,8 @@
     &:hover {
         cursor: pointer;
         .docs__poster--play-button {
-          color: $c-media-default;
-          animation-name: buttonSwitchIn;
-          animation-duration: .5s;
-          animation-fill-mode: forwards;
+          transform: scale(1.05);
```
Comment (mr-mr) [inline src/css/_youtube.scss]: Does this need to be turned off on mobile? Does hover not count as tap?

File: src/css/_youtube.scss [hunk 2]
```diff
@@ -113,55 +106,41 @@
...
+      margin-top: 14px;
+      padding-right: 7px;
+      padding-left: 8px;
+      margin-left: 4px;
+      border-left: 1px solid rgba(51,51,51,0.6);
+      display: inline-block;
+    }
+    @include icon-play;
   }
   &--yellow {
```
Comment (mr-mr) [inline src/css/_youtube.scss]: Not entirely sure this is needed, if you're moving the keyframes css

File: src/css/_youtube.scss [hunk 3]
```diff
@@ -113,55 +106,41 @@
...
     color: $c-media-default;
   }
 }
```
Comment (mr-mr) [inline src/css/_youtube.scss]: Not entirely sure this is needed, if you're moving the keyframes css

**Comment by akash1810** [comment]
this is a little old now, should we rebase and merge or close?

**Pull request merged**